### PR TITLE
Rebrand

### LINKED ISSUE #7: Switch to temporary meta server
New meta server `https://github.com/PlaceholderMC/meta-polymc`

### LINKED ISSUE #8: Replace files.polymc.org for fmllibs

### LINKED ISSUE #9: Branding

### LINKED ISSUE #12: Fork PolyMC/libnbtplusplus
Fork https://github.com/PolyMC/libnbtplusplus

### PR DESCRIPTION
**Pull request by Scrumplex**
# TODO
- [x] Move to new org
  - [x] Update references to PlaceholderMC

Closes #12
Closes #9 
Closes #8 
Closes #7

### CODE REVIEW

File: program_info/CMakeLists.txt [hunk 1]
```diff
@@ -8,44 +8,44 @@ if(UNIX)
     endif()
 endif()
 
-set(Launcher_CommonName "PolyMC")
+set(Launcher_CommonName "PrismLauncher")
 
-set(Launcher_Copyright "PolyMC Contributors\\n© 2012-2021 MultiMC Contributors")
+set(Launcher_Copyright "Prism Launcher Contributors\\n© 2012-2021 MultiMC Contributors")
```
Comment (txtsd) [inline program_info/CMakeLists.txt:13]: Don't we have to mention PolyMC Contributors anyway, and add Prism Launcher Contributors to it instead?

File: program_info/prismlauncher.rc.in [hunk 2]
```diff
@@ -15,10 +15,10 @@ BEGIN
         BEGIN
                 BLOCK "000004b0"
                 BEGIN
-                        VALUE "CompanyName", "MultiMC & PolyMC Contributors"
-                        VALUE "FileDescription", "PolyMC"
+                        VALUE "CompanyName", "MultiMC & Prism Launcher Contributors"
```
Comment (txtsd) [inline program_info/prismlauncher.rc.in:18]: Ditto

File: program_info/CMakeLists.txt (hunk 1 above)
Comment (Scrumplex) [reply program_info/CMakeLists.txt:13]: Prism Launcher contributors = PolyMC contributors

Comment (txtsd) [reply program_info/CMakeLists.txt:13]: Because the license is the same?

Comment (mrmelon54) [reply program_info/CMakeLists.txt:13]: I would also agree the true PolyMC contributors are all moving to Prism Launcher

**Comment by Steveplays28** [comment]
The issue templates also need to be updated

And you might also want to change the OpenCollective link in the "Sponsor this project" section to point to the new OpenCollective link (https://opencollective.com/prismlauncher)

File: program_info/CMakeLists.txt (hunk 1 above)
Comment (Sascha-T) [reply program_info/CMakeLists.txt:13]: > Prism Launcher contributors = PolyMC contributors

Can we get a legal-ish opinion in here on what to actually do?

**Comment by TheAirBlow** [comment]
@Scrumplex change the Curseforge API key too.

File: program_info/CMakeLists.txt (hunk 1 above)
Comment (Scrumplex) [reply program_info/CMakeLists.txt:13]: Legally, it doesn't matter. The copyright headers in the code matter. This is just a text that is shown in some places

Comment (vijfhoek) [reply program_info/CMakeLists.txt:13]: By that logic the MultiMC contributors shouldn't need to be mentioned either right?

**Comment by Bildcraft1** [comment]
immagine
There is still the PolyMC license in the About Screen

File: program_info/CMakeLists.txt (hunk 1 above)
Comment (Scrumplex) [reply program_info/CMakeLists.txt:13]: Yes. But I am fine with MultiMC people

**Comment by Scrumplex** [comment]
> immagine There is still the PolyMC license in the About Screen

fixed

**Comment by vijfhoek** [comment]
There's also still PolyMC stuff in the `nix/` subfolder, but I imagine it's better to wait until the rebrand is done and all websites are set up before tackling that

EDIT: I accidentally skipped over the changes in the PR and you're already tackling the default.nix. Guess the other files could be tackled as well right away then

**Comment by Scrumplex** [comment]
I didn't touch most markdown files, as they are not really important right now

**Comment by Bildcraft1** [comment]
For now the report bug button links goes to an 404 Github Page, but it will probably change when the Github Organization will be renamed in "PrismLauncher" i guess

**Pull request merged**